GOVERNANCE.md

```diff
@@ -148,10 +148,12 @@ After the nominee's first prepared release has been promoted the new releaser mu
 * Be added to the GitHub [releasers team](https://github.com/orgs/nodejs/teams/releasers) in the Node.js org (grants ci-release access)
 * Be added to the GitHub [security-release team](https://github.com/orgs/nodejs/teams/security-release) in the Node.js and nodejs-private orgs
 * Have a single, high quality SSH key added to the "dist" user on the primary www server (see below for guidelines regarding SSH key quality)
-* Have a GPG key added to the nodejs/node [README.md](https://github.com/nodejs/node/#release-team)
-  * The key should be cherry-picked to all active release branches.
-  * Any future updates to the key should also be cherry-picked to all active release branches.
-* Open a PR in [nodejs/docker-node](https://github.com/nodejs/docker-node/) to add gpg key to [node.keys](https://github.com/nodejs/docker-node/blob/master/keys/node.keys)
+* Open a PR to add their GPG key to the nodejs/node [README.md](https://github.com/nodejs/node/#release-team)
```

Comment:
What anchor is the README link supposed to link to? I don't think that anchor exists anymore. Maybe it needs to be https://github.com/nodejs/node/#release-keys?
```suggestion
* Open a PR to add their GPG key to the nodejs/node [README.md](https://github.com/nodejs/node/#release-keys)
```